__tests__/app.test.js

```diff
@@ -186,10 +178,204 @@ describe("ARTICLES", () => {
     })
   });
 
+     test("Returns the 400 status code when given an invalid article ID", () => {
+      return request(app)
+      .get('/api/articles/number-one')
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe('Bad Request')
+      })
+    })
+  });
+
 })
 
 
 
+
+describe("QUERY", () => {
+
+
+
+
+  //This needs sorting.
+  test("GET: Returns 200 status code and an array sorted by the created_at (date) by default. ", () => {
+
+    return request(app)
+    .get('/api/articles')
+    .expect(200)
+    .then((response) => {
+
+      const articles = response.body.articles;
+
+      expect(articles).toBeSortedBy('created_at', { descending: true });
+
+    })
+  });
+
+
+
+
+  test("GET: Returns 200 status code and an array sorted by 'author' if this is the given query.", () => {
+
+    return request(app)
+    .get('/api/articles?sort_by=author&order=asc')
+    .expect(200)
+    .then((response) => {
+
+      const articles = response.body.articles;
+
+      expect(articles).toBeSortedBy('author', { descending: false })
+
+    })
+  });
+
+
+
+
+  test("GET: Returns 200 status code and an array sorted by 'title' if this is the given query.", () => {
+
+    return request(app)
+    .get('/api/articles?sort_by=title&order=asc')
+    .expect(200)
+    .then((response) => {
+
+      const articles = response.body.articles;
+
+      expect(articles).toBeSortedBy('title', { descending: false })
+    })
+  });
+
+
+
+
+  test("GET: Returns 200 status code and an array sorted by 'article_id' if this is the given query.", () => {
+
+    return request(app)
+    .get('/api/articles?sort_by=article_id&order=asc')
+    .expect(200)
+    .then((response) => {
+
+      const articles = response.body.articles;
+
+      expect(articles).toBeSortedBy('article_id', { descending: false })
+    })
+  });
+
+
+
+
+  test("GET: Returns 200 status code and an array sorted by 'topic' if this is the given query.", () => {
+
+    return request(app)
+    .get('/api/articles?sort_by=topic&order=asc')
+    .expect(200)
+    .then((response) => {
+
+      const articles = response.body.articles;
+
+      expect(articles).toBeSortedBy('topic', { descending: false })
+    })
+  });
+
+
+
+
+  test("GET: Returns 200 status code and an array sorted by 'created_at' if this is the given query.", () => {
+
+    return request(app)
+    .get('/api/articles?sort_by=created_at&order=asc')
+    .expect(200)
+    .then((response) => {
+
+      const articles = response.body.articles;
+
+      expect(articles).toBeSortedBy('created_at', { descending: false })
+    })
+  });
+
+
+
+
+  test("GET: Returns 200 status code and an array sorted by 'votes' if this is the given query.", () => {
+
+    return request(app)
+    .get('/api/articles?sort_by=votes&order=asc')
+    .expect(200)
+    .then((response) => {
+
+      const articles = response.body.articles;
+
+      expect(articles).toBeSortedBy('votes', { ascending: true })
```

Comment:
I would stick to calling tis descending:false like you have in the other tests with order=asc